independent-publisher-connectors/Shopify Admin Content/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2895 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Shopify Admin Content",
+    "description": "The Shopify REST Admin API lets you build apps and other integrations for the Shopify admin.",
+    "contact": {
+      "name": "Hitachi Solutions",
+      "email":"jlindstrom@hitachisolutions.com"
+    }
+  },
+  "host": "thatapiguy.myshopify.com",
+  "basePath": "/admin/api/2021-04/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/blogs/{blog_id}/articles.json": {
+      "get": {
+        "description": "Retrieves a list of all articles from a certain blog",
+        "summary": "Get a list of all articles of a blog",
+        "tags": [
+          "Article"
+        ],
+        "operationId": "ListBlogArticles",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "blog_id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The ID of the blog containing the articles.",
+            "x-ms-summary": "Blog ID",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
```

Comment:
@thatapiguy Empty strings are not allowed for any description value.

Reply:
Added descriptions.